Makefile

```diff
@@ -16,7 +16,6 @@ proto-py:
 
 proto-py-pkg:
 	python -m grpc_tools.protoc -I protos --python_out=modelzoo/_protos --grpc_python_out=modelzoo/_protos protos/services.proto
-
```

Comment:
?

Reply:
Ah I wrote a proto-py-pkg thing for model_io and then saw that you had written one too, above, so I deleted it. Must have accidentally deleted a line as well.